docs/pricing/am2-pricing.mdx

```diff
@@ -0,0 +1,262 @@
+---
+title: Pricing & Billing (Before June 11, 2024)
```

Comment:
```suggestion
title: Pricing & Billing (before June 11, 2024)
```
I know it goes against convention, but I think it looks better to have this lowercased